### ISSUE TITLE
Fieldset missing on form

### ISSUE THREAD

**Issue by jamestyack** [UnlockedLab/unlockphilly#45]
FYI, I ran the new link through Wave accessibility and it highlighted an issue with the answer options with the following, “Missing Fieldset; a group of check boxes or radio buttons are not enclosed within a fieldset”. So, I don’t know what that means or anything, but it sounds important. What I don’t know could fill a canyon. Here is the link for Wave: http://wave.webaim.org/.   (Faith's email)
